### ISSUE TITLE
When trying to addStream() yPosition not working

### ISSUE THREAD

**Issue by Iulian33** [t-mullen/video-stream-merger#73]
```
 const merger = new VideoStreamMerger();
    merger.addStream(mediaStream, {
      x: 200,
      y: 410,
      width: 400,
      height: 225,
      mute: true
    });
```
not working `y:410` in version `@4.0.0`

**Comment by hthetiot** [comment]
I will check, thx @Iulian33

**Comment by hthetiot** [comment]
I did not saw the difference between V3 implementation, initially then saw my typo "stream.Y" vs "stream.y".
Sorry will make PR to fix ASAP.

```js
      // default draw function
      const width = stream.width == null ? this.width : stream.width
      const height = stream.height == null ? this.height : stream.height
      this._ctx.drawImage(stream.element, stream.x, stream.y, width, height)
```
Source: https://github.com/t-mullen/video-stream-merger/blob/0cfc83c382ab39ca3ae81f18a11424ce4c650af0/src/index.js#L338

```js
    const canvasHeight = this.height;
    const canvasWidth = this.width;

    const height = stream.height || canvasHeight;
    const width = stream.width || canvasWidth;

    let positionX = stream.x || 0;
    let positionY = stream.Y || 0;

    try {
        this._ctx?.drawImage(element, positionX, positionY, width, height);
    } catch (err) {
      // Ignore error possible "IndexSizeError (DOM Exception 1): The index is not in the allowed range." due Safari bug.
      console.error(err);
    }
```
Source: https://github.com/t-mullen/video-stream-merger/blob/9edd4057d8f6564011d821e30bde4a7b4db67e37/src/index.ts#L535

Note: the V4 is a bit different cause had KeepRatio implementation in initial PR, will make separate PR for stream.keepRatio

**Comment by t-mullen** [comment]
Fixed in 4.0.1

**Comment by Iulian33** [comment]
Thx a lot ! appreciate  your effort
